Milestone 1 : Portfolio accessibility pair programming

joseheco opened the pull request:
This is the result of the accessibility for us:

- Page titles - no issues found.

- Image text alternatives – added an appropriate description to all project images.

- Headings – we changed the ‘get started’ section to an h2.

- Color contrast – no issues found.

- Resize text – no issues found.

- Interaction – we added shadow to the input elements when they have focus.

- Moving content – We suggest that you could use the article tag to enclose the projects section.

mikethreels reviewed (changes requested):
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
Great job on adding the report to the description of this PR 💪 however according to the requirements, If issues were found, they should link to the corresponding line of code in the Pull Request, read [this article](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/) on how to do that and it should look like the below example:
https://github.com/joseheco/My_Portfolio/blob/152ebc69184a21e24577e15d06fa6243aa32b830/index.html#L43

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

williamrolando88 commented on style.css:
```diff
@@ -1085,3 +1060,6 @@ footer button h4{
   color: white;
 }
 
+input:focus {
```
Interaction – we added a shadow to the input elements when they have focus.

williamrolando88 commented on index.html:
```diff
@@ -1,215 +1,330 @@
+          <img
+            class="lap-images_1"
+            src="./images/laptop_1.png"
+            alt="laptop-project-1"
```
Image text alternatives – added an appropriate description to all project images.

williamrolando88 commented:
Color contrast – no issues found.

williamrolando88 commented:
Hi, @mikethreels ✋ 

Is this the right way to do the references to the changes we already made to the project? I'm asking this because they look just a little different.

Thank you for your help 😄

joseheco commented:
This is the result of the accessibility for us:

- Page titles - no issues found.

- Image text alternatives – added an appropriate description to all project images.

https://github.com/joseheco/My_Portfolio/blob/8adc45cf58347900f667625063f6c4b6df9e13c9/index.html#L75-L79

- Headings – we changed the ‘get started’ section to an h2.

https://github.com/joseheco/My_Portfolio/blob/8adc45cf58347900f667625063f6c4b6df9e13c9/index.html#L242

- Color contrast – no issues found.

- Resize text – no issues found.

- Interaction – we added shadow to the input elements when they have focus.

https://github.com/joseheco/My_Portfolio/blob/152ebc69184a21e24577e15d06fa6243aa32b830/style.css#L1063-L1065

- Moving content – We suggest that you could use the article tag to enclose the projects section.

https://github.com/joseheco/My_Portfolio/blob/152ebc69184a21e24577e15d06fa6243aa32b830/index.html#L69-L191

mikethreels reviewed (approved):
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
